### ISSUE TITLE
Show date wishlist item is added

### ISSUE THREAD

**Issue by dtwilliamson** [aunefyren/poenskelisten#51]
If wishlist items were to show the date they were added it would aid shoppers/gift-givers in determining how current a wish is. For example a wish that's a couple of years old may no longer be wanted or the wisher may have bought it themselves. In either case they may have forgotten to remove the item. The shopper can make a judgement call using the date.

Also the wisher can use it for themselves to help consider whether they want to remove it.

**Comment by aunefyren** [comment]
Agreed, the date should probably be updated when the wish is edited as well. I also think the wishlist should show when it was last updated.

**Comment by aunefyren** [comment]
Implemented in [v2.1.0](https://github.com/aunefyren/poenskelisten/releases/tag/v2.1.0).
